lib/filesystem/CFilesystemLoader.h

```diff
@@ -50,7 +51,7 @@ class DLL_LINKAGE CFilesystemLoader : public ISimpleResourceLoader
 	 * key = ResourceID for resource loader
 	 * value = name that can be used to access file
 	*/
-	std::unordered_map<ResourceID, boost::filesystem::path> fileList;
+	mutable std::unordered_map<ResourceID, boost::filesystem::path> fileList;
```

Comment:
Adding "mutable" here because fileList is merely the cache and not the real instance state (over which we do not have the control - it is volatile and belongs to OS).